File: Dddml.Wms.HttpServices.PhpClient/Wms/Domain/Metadata/BoundedContextMetadata.php
<?php

namespace Wms\Domain\Metadata;

class BoundedContextMetadata
{
    const REQUESTER_ID = 'requesterId';
    const COMMAND_TYPE = 'commandType';
    const COMMAND_ID   = 'commandId';

    const HTTP_SERVICE_ORDERS_QUERY_NAME = 'sort';
    const HTTP_SERVICE_FIRST_RESULT_QUERY_NAME = 'firstResult';
    const HTTP_SERVICE_MAX_RESULTS_QUERY_NAME = 'maxResults';
    const HTTP_SERVICE_RETURNED_FIELDS_QUERY_NAME = 'fields';
    const HTTP_SERVICE_FILTER_QUERY_NAME = 'filter';

}

File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/meta/BoundedContextMetadata.java
package org.dddml.wms.domain.meta;

public interface BoundedContextMetadata
{
    String REQUESTER_ID = "requesterId";
    String COMMAND_TYPE = "commandType";
    String COMMAND_ID   = "commandId";

    String HTTP_SERVICE_ORDERS_QUERY_NAME = "sort";
    String HTTP_SERVICE_FIRST_RESULT_QUERY_NAME = "firstResult";
    String HTTP_SERVICE_MAX_RESULTS_QUERY_NAME = "maxResults";
    String HTTP_SERVICE_RETURNED_FIELDS_QUERY_NAME = "fields";
    String HTTP_SERVICE_FILTER_QUERY_NAME = "filter";

}


